modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImpl.java

```diff
@@ -286,6 +297,14 @@ protected void activate() {
 			portalCacheManager = _multiVMPool.getPortalCacheManager();
 
 		portalCacheManager.registerPortalCacheManagerListener(this);
+
+		_serviceRegistration = bundleContext.registerService(
```

Comment:
Hey @GabrielHenriquedev, please put `_serviceRegistration`  after the `_dbPartitionEnabled`, thanks.